### DISCUSSION TITLE
What happened to `EbookUI` in 3.4?

### DISCUSSION THREAD

**Discussion by tony** [sumatrapdfreader/sumatrapdf#2767] [Q&A]
in re: https://github.com/sumatrapdfreader/sumatrapdf/discussions/2595#discussioncomment-2842401

What exactly happened to epub reading 3.4?

I am surprised since epub reader with `UseFixedPageUI` was one of the best ebook viewers I've used.

Are there any comparable epub alternatives out there until sumatra supports it again?

<details>

<summary>Example of a config possible in 3.3</summary>

```
EbookUI [
    FontName = Roboto
    FontSize = 12
    UseFixedPageUI = true
]
```

</details>

**Comment by mistune** [comment]
Is 3.3 the last version that supports EbookUI settings? I was also looking for the EbookUI setting but had to manually add it. After restarting sumatra the setting gets pushed down and says its not supported in this version.
